lib/human_player.rb

```diff
@@ -0,0 +1,114 @@
+=begin
+Winning patterns
+  #horizontal wins
+  [0][0], [0][1], [0][2]
+  [1][0], [1][1], [1][2]
+  [2][0], [2][1], [2][2]
+
+  #vertical wins
+  [0][0], [1][0], [2][0]
+  [0][1], [1][1], [2][1]
+  [0][2], [1][2], [2][2]
+
+  #diagonal wins
+  [0][0], [1][1], [2][2]
+  [0][2], [1][1], [2][0]
+=end
+
+class Board
+    def initialize
+      @boxNumbers = [
+        ["1", "2", "3"],
+        ["4", "5", "6"],
+        ["7", "8", "9"]
+      ]
+      @top_row = "| | | |"
+      @mid_row = "| | | |"
+      @bot_row = "| | | |"
+  
+      @rules = ["|1|2|3|", "|4|5|6|", "|7|8|9|"]
+    end
+  
+    def show
+      puts @top_row
+      puts @mid_row
+      puts @bot_row
+    end
+  
+    def show_rules
+      puts "Welcome to Tic Tac Toe!"
+      puts "These are the diffenrent move positions you can make:"
+      puts @rules, ""
+    end
+  
+    def check(a, b, player)
+      @boxNumbers[a][b] == player
+    end
+  
+    def move(position, player)
+      case position
+      when "1"
+        @boxNumbers[0][0] = player
+        @top_row[1] = player
+      when "2"
+        @boxNumbers[0][1] = player
+        @top_row[3] = player
+      when "3"
+        @boxNumbers[0][2] = player
+        @top_row[5] = player
+      when "4"
+        @boxNumbers[1][0] = player
+        @mid_row[1] = player
+      when "5"
+        @boxNumbers[1][1] = player
+        @mid_row[3] = player
+      when "6"
+        @boxNumbers[1][2] = player
+        @mid_row[5] = player
+      when "7"
+        @boxNumbers[2][0] = player
+        @bot_row[1] = player
+      when "8"
+        @boxNumbers[2][1] = player
+        @bot_row[3] = player
+      when "9"
+        @boxNumbers[2][2] = player
+        @bot_row[5] = player
+      end
+    end
+  
+    def win(player)
+      #horizontal wins
+      return true if (0..2).all? { |i| check(0, i, player) }
+      return true if (0..2).all? { |i| check(1, i, player) }
+      return true if (0..2).all? { |i| check(2, i, player) }
+      #vertical wins
+      return true if (0..2).all? { |i| check(i, 0, player) }
+      return true if (0..2).all? { |i| check(i, 1, player) }
+      return true if (0..2).all? { |i| check(i, 2, player) }
+      #diagonal wins
+      return true if (0..2).all? { |i| check(i, i, player) }
+      return true if (0..2).all? { |i| check(i, 2 - i, player) }
+    end
+  
+  end
+  
+  
+  #run game
+  ttt = Board.new
+  player = ["x", "o"]
+  
+  ttt.show_rules
+  
+  loop do
+    puts "It's Player #{player[0]}'s turn."
+    input = gets.chomp
+    ttt.move input, player[0]
+    ttt.show
+    break if ttt.win player[0]
+    player[0], player[1] = player[1], player[0]
+  end
+  
+  puts "Congrats, Player #{player[0]} you win 🎉🎉"
```

Comment:
This block of code to me does not belong in this file. It is the main game loop, and should be moved to a file that better represents it.